feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/BookmarkScreen.kt

```diff
@@ -38,7 +38,9 @@ fun BookmarkScreen(
         uiState = uiState,
         onBackPressClick = onBackPressClick,
         onTimetableItemClick = onTimetableItemClick,
-        onBookmarkClick = viewModel::updateBookmark,
+        onBookmarkClick = { timetableItem, _ ->
+            viewModel.updateBookmark(timetableItem)
```

Comment:
I would like to use "onBookmarkClick" as the name of this method. It is an existing implementation though 🙇